src/react-components/room/TwitterOAuthModalContainer.js

```diff
@@ -30,7 +30,14 @@ export function TwitterOAuthModalContainer({ hubChannel, onConnected, onClose })
         popup.focus();
         popupRef.current = popup;
       } catch (error) {
-        console.error(error);
+        if (error === "twitter_probably_misconfigured") {
```

Comment:
This seems wrong now that I look at it. `error` should probably be an `Error` object, not a string.